### DISCUSSION TITLE
Jetsonnano & yolort

### DISCUSSION THREAD

**Discussion by ONNONS** [zhiqwang/yolov5-rt-stack#333] [Q&A]
Hello, I am trying to create an engine file for deepstram 6.0 in jetsonano through yolort. How can I do it? How to make an engine for jetsonano.

And when is the colab coming out?

**Comment by zhiqwang** [comment]
Hi @ONNONS ,

I didn't have experience in using Jetson, but as far as I know, it should be possible to use yolort if you could upgrade TensorRT to 8.2 in Jetson. Contribution about using yolort for Jetson is very welcome!

Check the TensorRT tutorial here: https://zhiqwang.com/yolov5-rt-stack/notebooks/onnx-graphsurgeon-inference-tensorrt.html and C++ usage tips here: https://github.com/zhiqwang/yolov5-rt-stack/tree/main/deployment/tensorrt#usage .

> And when is the colab coming out?

Hope to add the Colab today, and seems that we need at least a Colab pro account to be able to use TensorRT, so I wouldn't add the TensorRT's usage notebooks to Colab at this time.

**Comment by zhiqwang** [comment]
Just FYI @ONNONS , JetPack 4.6.1 includes [TensorRT 8.2.1](https://docs.nvidia.com/deeplearning/tensorrt/release-notes/tensorrt-8.html#rel-8-2-1) is released, see https://developer.nvidia.com/embedded/jetpack for more details.

**Comment by triple-mu** [comment]
We can get more information from https://developer.nvidia.com/jetpack-sdk-461.
